redash/models.py

```diff
@@ -431,9 +580,16 @@ class User(TimestampMixin, db.Model, BelongsToOrgMixin, UserMixin, PermissionsCh
                      unique=True)
 
     disabled_at = Column(db.DateTime(True), default=None, nullable=True)
+    details_default = {u'active_at': None}
+    details = Column(MutableDict.as_mutable(postgresql.JSON), nullable=True,
+                     server_default=json_dumps(details_default),
+                     default=details_default)
+    active_at = json_cast_property(db.DateTime(True), 'details', 'active_at', default=None)
```

Comment:
Doesn't having default value set for `active_at` enough so we don't have to set a default value in the `details` object?

Reply:
Good question, lemme check.

Reply:
Yep, fixed now.